pkg/grafana/dashboard-handler.go

```diff
@@ -56,13 +57,23 @@ func (h *DashboardHandler) GetExtension() string {
 	return "json"
 }
 
-func (h *DashboardHandler) newDashboardResource(path, uid, filename string, board Dashboard) grizzly.Resource {
+// APIVersion returns the api version for this resource
+func (h *DashboardHandler) APIVersion() string {
+	return "grafana.com/grizzly/v1"
```

Comment:
What do you think to use a more k8s style for API version?
 ```suggestion
	return "grizzly.grafana.com/v1"
```

Reply:
Is that more common? I’m not an expert there.

Reply:
@malcolmholmes yes, see (as example):
 - `ingress` that use `networking.k8s.io/v1 ` https://kubernetes.io/docs/concepts/services-networking/ingress/
 - `CRD` that use `apiextensions.k8s.io/v1 ` https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/

Reply:
Thanks for this suggestion. Around the time of your suggestion I embedded this suggestion into https://github.com/grafana/jsonnet-libs/tree/master/grizzly, although in hindsight, it should be `grizzly.grafana.com/v1alpha1` not `v1`. I've used `v1alpha1` here. I should fix jsonnet-libs to match.